src/server/src/test/resources/openssl.cnf

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023-2023 DataStax Inc.
```

Comment:
Question: why do we need a whole openssl configuration here? Is there something that we can't control from the command line flags?

Reply:
Do we want to control it with command line flags instead of files? The openssl.cnf was from management-api repository, I wanted to ensure we generate the certificates the same way.

Reply:
I'm actually OK with this I think, leave it as is.